packages/office-ui-fabric-react/src/components/Facepile/Facepile.Props.ts

```diff
@@ -76,6 +76,11 @@ export interface IFacepilePersona extends React.HTMLProps<HTMLButtonElement | HT
   onMouseOut?: (ev?: React.MouseEvent<HTMLElement>, persona?: IFacepilePersona) => void;
 
   /**
+   * If provided, the handler will be called when the component is mounted or unmounted
+   */
+  refHandler?: (el?: HTMLElement, persona?: IFacepilePersona) => void;
+
```

Comment:
Since IFacepilePersona extends React.HTMLProps it already has ref as something that you can pass in. Does something need the persona?